projects/canopy/src/lib/carousel/carousel.component.scss

```diff
@@ -1,5 +1,61 @@
+@import '../../styles/mixins.scss';
+
 .lg-carousel {
   overflow: hidden;
+  position: relative;
+
+  &__autoplay {
+    position: absolute;
+    right: var(--space-sm);
+    z-index: 1;
+    top: var(--space-sm);
+
+    &__button {
+      background-color: var(--color-white);
+      border-radius: 50%;
+      border: none;
+      width: var(--space-xxl);
+      height: var(--space-xxl);
+      font-size: var(--text-fs-3-size);
+      cursor: pointer;
+
+      &[keyboard-focus]:focus {
```

Comment:
Canopy shouldn't handle the `keyboard-focus`. I think we are moving away from having focus only on keyboard so this should be changed to allow focus normally.

Reply:
Yes, I just saw in some places we are have it and then I used it ;)